mariadb_skysql/manifest.json

```diff
@@ -0,0 +1,57 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "bed39b01-bbc5-4819-9744-7cf4a3d90f6d",
+  "app_id": "mariadb-skysql",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor SkySQL MariaDB instances with Datadog.",
+    "title": "MariaDB SkySQL",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "MariaDB SkySQL Observability",
+        "image_url": "images/image1.jpg"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Category::Data Store",
+      "Offering::Integration"
```

Comment:
What types of data is this integration submitting to Datadog? Please specify with by adding the following:

`"Submitted Data Type::Metrics"`